R/pedigree.R

```diff
@@ -421,67 +550,23 @@ getAInv <- function(ped) {
 #' stopifnot(!any(abs(A - AExp) > .Machine$double.eps))
 #' stopifnot(Matrix::isSymmetric(A))
 getA <- function(ped, labs = NULL) {
+
+    L = relfactor(ped)
```

Comment:
@cheuerde we don't want this for a large pedigree - the `getASubset()` uses the indirect Colleau method that gives a subset of A without ever forming the full L matrix,